### PR TITLE
Angular / Web components SDK: Allow passing Data and Context, define outlet component for editable children in custom components

### PR DESCRIPTION
**Pull request by teleaziz**

### CODE REVIEW

**Comment by teleaziz** [comment]
https://www.loom.com/share/a68470a7d8224a08bcc9c910db6a1b27

File: packages/angular/README.md [hunk 1]
```diff
@@ -55,6 +55,37 @@ Simply replace your 404 component with something like the below to allow creatin
...
+contentLoaded(data) {
+  // Data object (via the output $event) includes your custom fields, e.g. if you have a custom field named
+  // "title"
+  document.title = data.data.title
+}
+```
+
+## Builder sections within existing pages
+
+With setction models you can also use Builder.io components in/around existing pages (aka it doesn't have to control the whole page). See info on making custom models for this [here](https://www.builder.io/c/docs/guides/getting-started-with-models)
```
[the remark this answers is not in the document]
Comment (mrkoreye) [reply packages/angular/README.md:79]: Also, I saw from the loom that you wanted to document how to pass data down (`[data]="data"`), did you add that somewhere to the readme? I didn't see it

File: packages/angular/src/app/modules/builder/components/builder-component/builder-component.component.ts [hunk 2]
```diff
@@ -122,8 +125,9 @@ export class BuilderComponentComponent implements OnDestroy, OnInit {
       return null;
     }
     const script = document.createElement('script');
-
-    const wcVersion = getQueryParam(location.href, 'builder.wcVersion');
+    // TODO remove hardcoded version, maybe a tag?
+    const ANGULAR_LATEST_VERSION = '1.3.47-9';
```
Comment (mrkoreye) [inline packages/angular/src/app/modules/builder/components/builder-component/builder-component.component.ts:129]: Do we plan on removing this after doing a webcomponents deploy?

File: packages/react/src/components/builder-blocks.component.tsx [hunk 3]
```diff
@@ -145,4 +147,32 @@ export class BuilderBlocks extends React.Component<BuilderBlocksProps, BuilderBl
       </TagName>
     );
   }
+
+  static renderInto(
+    elementOrSelector: string | HTMLElement,
+    props: BuilderBlocksProps = {},
+    builderState: any
+  ) {
+    console.debug('BuilderBlocks.renderInto', elementOrSelector, props, this);
```
Comment (mrkoreye) [inline packages/react/src/components/builder-blocks.component.tsx:156]:
```suggestion
```

**Review by mrkoreye — APPROVED** [review]

**Comment by EasonCo** [comment]
When can we expect this in a stable release? the<builder-blocks-component> is particularly needed for Angular projects that want to use 'Components Only' mode or, more generally, have greater control over the layout from a custom component perspective.

**Comment by teleaziz** [comment]
@EasonCo this is currently released on @2.1.0 , please load the latest SDK and let us know of any questions

**Pull request merged**